### ISSUE TITLE
Massive renaming and removing of useless things

### ISSUE THREAD

**Issue by akuleshov7** [saveourtool/save-cloud#2904]
We should do a lot of renaming and removal of legacy shit. 

1. Rename all packages: 
    a. remove `save` and `save-cloud` prefixes completely for common modules. It can only be:  `save/cosv-common` (ONLY for SAVE/COSV specific common, in terms of Kotlin MPP for Native + JVM + JS), `common` (MPP for JS + JVM which are used EVERYWHERE), `common-frontend` (JS **only** code shared between COSV, SAVE, etc.) and `common-backend` (JVM **only** logic shared between COSV, SAVE, if needed). 
    b. we should have `cosv-frontend`, `save-frontend` and `save-backend`, `cosv-backend`. For save-specific components it is ok to have `save-orchestrator`, `save-agent`, etc.;
    c. rename packages: COSV and common packages should NOT have `save` in it's name
        `save-cloud-common` packages should be renamed to `com.saveourtool.common`
3. Remove sandbox service (if someone wishes to have - can save in separate repository later);
4. After the removal of sandbox service - merge `orchestrator-common` with `save-orchestrator`;

**Comment by akuleshov7** [comment]
@nulls if you will have any other ideas - welcome to share

**Comment by nulls** [comment]
> @nulls if you will have any other ideas - welcome to share

1. Removing `sandbox` and merging `save-orchestator-common` with `save-orchestrator` can be done before renaming to reduce affected files
2. Suggestion to create a tag before renaming (or a branch at least)
3. We have `save-cloud-charts`, probably we need to split it too
4. What's about `api-gateway` and `authentication-service`? They are required for COSV and SAVE. Do we need to rename packages there?

**Comment by akuleshov7** [comment]
> 4. What's about `api-gateway` and `authentication-service`? They are required for COSV and SAVE. Do we need to rename packages there?

Yeah, need to rename their packages as well! `com.saveourtool.api-gateway`

**Comment by Cheshiriks** [comment]
We also need to rename the paths for entities in `save-cloud-common `
